test/corpus/attributes.txt

```diff
@@ -8,15 +8,15 @@ Global attributes
 --------------------------------------------------------------------------------
 
 (compilation_unit
-  (global_attribute_list
+  (global_attribute
     (attribute
       name: (identifier)))
-  (global_attribute_list
+  (global_attribute
     (attribute
       name: (identifier))
     (attribute
       name: (identifier)
-      (attribute_argument_list))))
+      (attribute_arguments))))
```

Comment:
These changes are going to be breaking and they're going to take us away from the Roslyn grammar definitions.

e.g. attribute_argument_list being wiped out and attribute_argument becoming attribute_arguments.

Originally I aligned with them because it made it easier to write the grammar but it's also been really helpful as the language evolves to see which rules they changed for the new features and how ours changes too. It's also provided a great way to know we're parsing the code correctly as we can match the definitive parser.

If moving away from that is desirable because it makes the grammar a lot smaller or enables some new tree-sitter functionality it might be worth the trade-off but if not I'd prefer to retain the alignment and compatibility.